javalib/src/main/scala/java/util/Properties.scala

```diff
@@ -14,6 +14,8 @@ class Properties(protected val defaults: Properties)
 
   def load(inStream: java.io.InputStream): Unit = ???
 
+  def load(reader: java.io.Reader): Unit = ???
```

Comment:
Please add `@stub` annotation.

Reply:
In general, are `@stub` suppose to be everywhere there is an empty implementation?

Reply:
Responded on gitter: :point_up: [March 21, 2019 12:20 PM](https://gitter.im/scala-native/scala-native?at=5c9373675547f774483ec225)